edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc

```diff
@@ -0,0 +1,102 @@
+  autoRollbackOnFailure: {}
+#    initMonitorTimeoutSeconds: 1800 <3>
+  extraManifests: <4>
+  - name: example-extra-manifests
+    namespace: openshift-lifecycle-agent
+  oadpContent: <5>
+  - name: oadp-cm-example
+    namespace: openshift-adp
+----
+<1> Defines the desired stage for the `ImageBasedUpgrade` CR. The value can be `Idle`, `Prep`, `Upgrade`, or `Rollback`.
+<2> Defines the target platform version, the seed image to be used, and the secret required to access the image.
+<3> (Optional) Specify the time frame in seconds to roll back when the upgrade does not complete within that time frame after the first reboot. If not defined or set to `0`, the default value of `1800` seconds (30 minutes) is used.
+<4> (Optional) Specify the list of `ConfigMap` resources that contain your custom catalog sources to retain after the upgrade, and your extra manifests to apply to the target cluster that are not part of the seed image.
+<5> Specify the list of `ConfigMap` resources that contain the OADP `Backup` and `Restore` CRs.
+
+include::modules/cnf-image-based-upgrade.adoc[leveloffset=+1]
+
+////
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
+
+* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent and GitOps ZTP]
+////
+
+include::modules/cnf-image-based-upgrade-guidelines.adoc[leveloffset=+1]
+
+////
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.adoc#cnf-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent]
+
+* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-for-sno[Performing an image-based upgrade with Lifecycle Agent and GitOps ZTP]
+
+* xref:../../edge_computing/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-hub-cluster[Preparing the hub cluster for ZTP]
+
+* xref:../../edge_computing/image_based_upgrade/cnf-preparing-for-image-based-upgrade.adoc#ztp-image-based-seed-image-config_preparing-for-image-based-upgrade[Seed image configuration]
+
+* xref:../../edge_computing/image_based_upgrade/cnf-preparing-for-image-based-upgrade.adoc#cnf-image-based-upgrade-creating-backup-resources_preparing-for-image-based-upgrade[Creating ConfigMap objects for the image-based upgrade with Lifecycle Agent]
+
+* xref:../../edge_computing/image_based_upgrade/cnf-preparing-for-image-based-upgrade.adoc#ztp-image-based-upgrade-creating-backup-resources-with-ztp_preparing-for-image-based-upgrade[Creating ConfigMap objects for the image-based upgrade with GitOps ZTP]
+
+* xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc[About installing OADP]
+////
```

Comment:
These xrefs are in a comment block and are not currently rendering. They will need to be re-reviewed when the rest of the content merges.